src/shared/components/palette/fixed-list/write-fixed-list-field.component.ts

```diff
@@ -7,18 +7,10 @@ import { AbstractFieldWriteComponent } from '../base-field/abstract-field-write.
   templateUrl: './write-fixed-list-field.html'
 })
 export class WriteFixedListFieldComponent extends AbstractFieldWriteComponent implements OnInit {
-
-  fixedListControl: FormControl;
-
   ngOnInit() {
-    let notEmpty = this.caseField.value !== null && this.caseField.value !== undefined;
-    this.fixedListControl = this.registerControl(new FormControl(notEmpty ? this.caseField.value : ''));
-    if (this.hasSanitisedDynamicListData()) {
-      this.fixedListControl.setValue(this.caseField.value);
+    let isNull = this.caseField.value === undefined || this.caseField.value === '';
+    if (isNull) {
+      this.caseField.value = null;
     }
   }
-
-  private hasSanitisedDynamicListData() {
-    return this.fixedListControl.value && this.fixedListControl.value.value;
```

Comment:
this line seems redundant to me so it should be ok to remove. But just for piece of mind can we verify dynamic lists are still working?

Reply:
ok.. Will ask Ashley to test dynamic lists as well to see this change